store/player.ts

```diff
@@ -23,20 +23,28 @@ export const usePlayerStore = defineStore(
       }
     }
 
-    function resetGame(mode: 'tryAgain' | 'exit' = 'exit') {
+    function resetGame(mode: 'tryAgain' | 'exit' | 'resetGame' = 'resetGame') {
       const { resetStages } = useStageStore()
+      if (mode === 'resetGame') {
+        playerHealth.value = 0
+        router.push('/')
+      }
```

Comment:
I don't know how your code works. how does the playerHealth value affects the route?

when the playerHealth is 100 I do not can push to the options (quit or try again) route.

Reply:
If playerHealth is 100, I can reset or restart the game without any problem
![chrome_zRfIFiU6xf](https://github.com/HDenizD/super-kung-vue/assets/27643142/37573ca3-1d85-48a1-ac5c-b0d6c65c82e0)